ms2/src/org/labkey/ms2/Protein.java

```diff
@@ -510,6 +541,15 @@ public void setPeptides(String... peptides)
         int countUnmodifiedPeptides;
         Map<String , Integer> countModifications;
         int countInstances;
+        @Getter @Setter String intensityColor;
+        @Getter @Setter Double intensity;
+        @Getter @Setter String confidenceColor;
+        @Getter @Setter Double confidence;
+        @Getter @Setter String foregroundColor;
+        @Getter @Setter boolean isIntensityView;

```

Comment:
Why is this a peptide-level setting? We'd never be showing intensity for some and confidence for others, would we?

Reply:
removed